src/common/utils.tsx

```diff
@@ -0,0 +1,11 @@
+export const filterObject = (data: any[], keys: string[]): any[] => {

```

Comment:
Could you add a docstring and/or comments to explain the purpose of this function? It's unclear with abstract variable names like `data`, `keys`, and `newObjArr`.

Reply:
Done: 3fb07cd